src/west/app/project.py

```diff
@@ -640,6 +644,48 @@ def _die_if_manifest_project_filter(self, action):
                      'the manifest while projects are made inactive by the '
                      'project filter.')
 
+    def _status(self):
+        topdir = Path(self.topdir)
+        ppaths = []
+        untracked = []
+        for project in self._projects(None):
+            # Skip inactive projects unless the user said
+            # --all or named some projects explicitly.
+            if not self.manifest.is_active(project):
+                self.dbg(f'{project.name}: skipping inactive project')
+                continue
```

Comment:
Is this desired? Now inactive projects will be printed as untracked, AFAICS?

Reply:
right, the problem is that with the APIs I am using the folder needs to be present on disk for it to work. I will think about it and provide something better.